app/javascript/react/components/MonstersIndex.js

```diff
@@ -1,5 +1,7 @@
-import React, { useState, useEffect} from "react"
-import MonsterTile from "./MonsterTile"
+import React, { useState, useEffect} from "react";
+import { Link } from 'react-router-dom';
```

Comment:
It looks like `Link` isn't being used here so we can remove this line

Reply:
Removed